packages/randii/randii.1.0.0/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "A pure OCaml port of the Random123 counter based random number generator from DEShaw Research"
+description:
+  "Currently implements the Threefry 2&4 digit 32&64 bit random number generators"
+maintainer: ["simon.parry@karoshibee.com"]
+authors: ["Simon Parry"]
+license: "MIT"
+homepage: "https://github.com/KaroshiBee/randii"
+doc: "https://github.com/KaroshiBee/randii"
+bug-reports: "https://github.com/KaroshiBee/randii/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "logs" {>= "0.7.0"}
+  "cmdliner" {>= "1.1.1"}
+  "fmt" {>= "0.9.0"}
+  "integers" {>= "0.4.0"}
+  "dune" {>= "3.0" & > "1.5"}
+  "testu01" {>= "1.2.3-0.2"}
```

Comment:
Is this expected to have an executable named `randii.u01_test` installed by this package? The README states that this package is a library and that the `randii.u01_test` is a "battery of tests"

Reply:
yes, I want to define an executable to run the test set defined in the ocaml package 'testu01'.  It shouldnt be run as part of the @runtest target (its too long running).

I think I'll add it as a subcommand of my main randii CLI - less confusing, apologies